lib/SILOptimizer/Analysis/RegionAnalysis.cpp

```diff
@@ -2346,15 +2349,16 @@ class PartitionOpTranslator {
   /// that define the block's dataflow.
   void translateSILBasicBlock(SILBasicBlock *basicBlock,
                               std::vector<PartitionOp> &foundPartitionOps) {
-    LLVM_DEBUG(llvm::dbgs() << SEP_STR << "Compiling basic block for function "
-                            << basicBlock->getFunction()->getName() << ": ";
-               basicBlock->dumpID(); llvm::dbgs() << SEP_STR;
-               basicBlock->print(llvm::dbgs());
-               llvm::dbgs() << SEP_STR << "Results:\n";);
+    REGIONBASEDISOLATION_LOG(
+        llvm::dbgs() << SEP_STR << "Compiling basic block for function "
+                     << basicBlock->getFunction()->getName() << ": ";
+        basicBlock->dumpID(); llvm::dbgs() << SEP_STR;
```

Comment:
Will this work without in both asserts and non-asserts?

https://github.com/swiftlang/swift/blob/d87be5a9db49b81d505328519bb2f5b25ee5cefc/include/swift/SIL/SILBasicBlock.h#L568 has `dumpID` inside a `#ifndef NDEBUG` block.